services/QuillLMS/engines/evidence/spec/lib/grammar/feedback_assembler_spec.rb

```diff
@@ -9,9 +9,9 @@ module Grammar
         let(:client_response) do
           {
             'highlight' => [{
-              'type': 'response',
-              'text': 'someText',
-              'character': 0
+              'type' => 'response',
```

Comment:
Note, I've found this confusing in the past (especially going back and forth from javascript and other languages), but `'key': ` creates a symbol key.
<img width="385" alt="Screen Shot 2022-08-19 at 5 14 36 PM" src="https://user-images.githubusercontent.com/1304933/185708301-34998f25-1614-436e-a061-29f30987c430.png">